content/en/cosign/timestamps.md

```diff
@@ -49,6 +49,27 @@ To verify, retrieve the TSA's certificate chain, which must contain the root CA
 cosign verify --timestamp-certificate-chain ts_chain.pem <artifact>
 ```
 
+### mTLS connection to the TSA server
+`cosign sign` accept several additional optional parameters to pass the CA certificate of
```

Comment:
Add full return after heading, subject / verb agreement 

```suggestion

`cosign sign` accepts several additional optional parameters to pass the CA certificate of
```

Reply:
done in d6dbe6d